packages/data/lib/repository/tmdb_repository.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:data/service/api_base_service.dart';
+import 'package:data/service/service_payload.dart';
+import 'package:data/utils/api_key_data.dart';
+import 'package:data/utils/constants.dart';
+import 'package:domain/model/tmdb_response.dart';
+import 'package:domain/repository/tmdb_repository.dart';
+
+class TMDBRepositoryImpl implements TMDBRepository {
+  final ApiBaseService<ServicePayload> _apiService;
+
+  TMDBRepositoryImpl(this._apiService);
+
+  @override
+  Future<TMDBResponse> getCastImage({required int? tMDBId}) async {
+    final response = await _apiService.get(
+      '${C.personUrl}/$tMDBId${C.imagesUrl}',
+      queryParameters: {Q.apiKey: ApiKeyData.apiKeyTMDB},
+      isTrakt: false,
+    );
```

Comment:
because I make one dio_builder for trakt and tmbd I need to show which one I need to use now. U can check logic in  file : https://github.com/elizaveta-seleznyova-idf/movie_and_stars/blob/feature/IO-514_details_page/packages/data/lib/service/api_service.dart